compiler/commands.nim

```diff
@@ -480,7 +480,7 @@ proc setCommandEarly*(conf: ConfigRef, command: string) =
   # must be handled here to honor subsequent `--hint:x:on|off`
   case conf.cmd
   of cmdRst2html, cmdRst2tex: # xxx see whether to add others: cmdGendepend, etc.
-    conf.foreignPackageNotes = {hintSuccessX}
+    conf.foreignPackageNotes = {hintSuccessX, hintBuildMode}
```

Comment:
this diff doesn't seem good; rst2html and rst2tex; -d:release etc seems irrelevant for those commands

Reply:
The theory here was again that successx becomes either that or buildmode, but in truth this one I wasn't quite sure of and almost asked about it.

Reply:
Changed it back.